src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -2255,6 +2256,97 @@ mesh::generate_strip(const conduit::Node& topo,
 }
 
 
+void
+mesh::generate_strip(const conduit::Node& topo,
+                     conduit::Node& topo_dest,
+                     conduit::Node& coords_dest,
+                     conduit::Node& fields_dest,
+                     conduit::Node& matsets_dest,
+                     const conduit::Node& options)
+{
+    const std::string topo_name = topo.name();
+    const std::string topo_dest_name = topo_dest.name();
+    std::string matset_prefix = "";
+    std::vector<std::string> matset_names;
+    const Node& matsets_src = (*(topo.parent()->parent()))["matsets"];
+
+    // check for existence of matset prefix
+    if (options.has_child("matset_prefix"))
+    {
+        if (options["matset_prefix"].dtype().is_string())
+        {
+            matset_prefix = options["matset_prefix"].as_string();
+        }
+        else
+        {
+            CONDUIT_ERROR("matset_prefix must be a string.");
+        }
+    }
+
+    // check for target matset names
+    if (options.has_child("matset_names"))
```

Comment:
Note: matset names could be inferred from the `field_names` options (or all fields).
I expect thats how folks would think to use it ("I want these fields, make it so")